Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7.16-alpine
+
+MAINTAINER coolboi567 <PrashantShahi567@gmail.com>
+
+WORKDIR /opt/dnstwister
+
+RUN apk update && apk add --virtual .build-deps gcc musl-dev && \
+    apk add git postgresql-dev && \
+    git clone https://github.com/thisismyrobot/dnstwister . && \
```

Comment:
Sure.
We can make use of build arguments or environment variables, but it has to be before the image is built, since, we are deleting the `.git` directory afterward to reduce the final image size.

Reply:
@thisismyrobot I have the new Dockerfile with build arguments ready.
But I think the right thing to do in Dockefile would be to copy the source code of the current directory with current files (irrespective of the git HEAD and/or any recent changes of any of the files).